action_utxo.go

```diff
@@ -97,7 +97,7 @@ func (c *Client) GetUtxo(ctx context.Context, xPubKey, txID string, outputIndex
 	var tx *Transaction
 	tx, err = getTransactionByID(ctx, "", utxo.TransactionID, c.DefaultModelOptions()...)
 	if err != nil {
-		c.Logger().Error(ctx, "failed finding transaction related to utxo: "+utxo.ID)
+		c.Logger().Error().Msg("failed finding transaction related to utxo: " + utxo.ID)
```

Comment:
Wow! It's a great example why we should use vanila zerolog :D

Now you can move, and add, information about utxo ID and transaction ID as context message like:

```
c.Logger().Error().Msg("failed finding transaction related to utxo").Str("utxo-id", utxo.ID).Str("tx-id", utxo.TransactionID)

Reply:
Good point, I added more parameters into logs. Mainly txID or utxo